generator/graphql-kotlin-federation/build.gradle.kts

```diff
@@ -6,7 +6,10 @@ plugins {
 
 dependencies {
     api(projects.graphqlKotlinSchemaGenerator)
-    api(libs.federation)
+    api(libs.federation) {
+        exclude(group = "com.graphql-java", module = "graphql-java")
+    }
+    api(libs.graphql.java)
```

Comment:
nit: since this is a dependency of schema generator you don't really need to have this explicit dependency (also gradle by default would pick the latest between two transitive dependency but yeah good to stick to the version from schema generator)